static/lang/re-en.json

```diff
@@ -3339,6 +3339,11 @@
                     },
                     "Single": "Single ikon with multiple copies."
                 },
+                "RedGoldMortality": {
+                    "DamageNote": "When you successfully damage an enemy with the ikon, a marking appears around the wound, painted in the red of mortal blood and the gold of divine ichor. When the target would regain Hit Points, it must attempt a Will save against your class DC to determine the effects, and then the marking fades. The marking otherwise fades after 1 minute. @UUID[Compendium.pf2e.feat-effects.Item.TpcfUI2y6Shg63V0]{Effect: Red-Gold Mortality}",
+                    "HealingNote": "<p>When the target would regain Hit Points, such as from a healing effect or an ability like fast healing or regeneration, it must attempt a @Check[will|traits:exemplar,ikon,void] save against your class DC to determine the effects, and then the marking fades. The marking otherwise fades after 1 minute.</p>\n<hr>\n<p><strong>Success</strong> The creature regains the full number of Hit Points that would be healed.</p>\n<p><strong>Failure</strong> The creature regains only half the number of Hit Points as the contradictory energies swirl within it.</p>\n<p><strong>Critical Failure</strong> The creature doesn't regain any Hit Points.</p>",
```

Comment:
```suggestion
                    "HealingNote": "<p>When the target would regain Hit Points, it must attempt a @Check[will|traits:exemplar,ikon,void] save against your class DC to determine the effects.",
```

Reply:
Without the results there isn't much point to this Note